dashbuilder/dashbuilder-runtime/src/main/java/org/dashbuilder/shared/service/RuntimeModelRegistry.java

```diff
@@ -16,7 +16,9 @@
 
 package org.dashbuilder.shared.service;
 
+import java.util.List;
```

Comment:
Looks like List is an unused import. You can remove if it is not required

Reply:
Thank you, I removed it in my last commit.